pkg/api/types.go

```diff
@@ -221,6 +224,17 @@ type GCEPersistentDisk struct {
 	ReadOnly bool `json:"readOnly,omitempty"`
 }
 
+// A ISCSI Disk can only be mounted as read/write once.
+type ISCSIDisk struct {
```

Comment:
Adding to that, iSCSI has some cloud affinity, it is the default implementation for OpenStack Cinder.